terraform/vars/terraform-dev.tfvars

```diff
@@ -1,10 +1,18 @@
-environment               = "dev"
-log_level                 = "debug"
-service_url               = "https://dev-data-api.globalforestwatch.org" # fake, needed for CloudFront
-rw_api_url                = "https://api.resourcewatch.org"
-desired_count             = 1
-auto_scaling_min_capacity = 1
-auto_scaling_max_capacity = 5
-lambda_analysis_workspace = "features-lat_lon"
-key_pair                  = "dmannarino_gfw"
-new_relic_license_key_arn = "arn:aws:secretsmanager:us-east-1:563860007740:secret:newrelic/license_key-lolw24"
+environment                    = "dev"
+log_level                      = "debug"
+service_url                    = "https://dev-data-api.globalforestwatch.org" # fake, needed for CloudFront
+rw_api_url                     = "https://api.resourcewatch.org"
+desired_count                  = 1
+auto_scaling_min_capacity      = 1
+auto_scaling_max_capacity      = 5
+lambda_analysis_workspace      = "features-lat_lon"
+key_pair                       = "dmannarino_gfw"
+create_cloudfront_distribution = false
+new_relic_license_key_arn      = "arn:aws:secretsmanager:us-east-1:563860007740:secret:newrelic/license_key-lolw24"
+load_balancer_security_group   = "sg-07c9331c01f8da1c8"
+load_balancer_arn              = "arn:aws:elasticloadbalancing:us-east-1:563860007740:loadbalancer/app/gfw-data-api-elb-shared-dev-lb/77f2fc64dcdf9ebc"
```

Comment:
was this just created via a feature branch? how do we make sure it doesn't get deleted by terraform/how can we create a new one if we update the ELB terraform code?

Reply:
This ELB is only associated with this branch's terraform workspace and code; other branches will just read this ARN. This one time I'll have to manually intervene to prevent a terraform destroy of this workspace when the branch is deleted.
If we want to change the ELB, we'd remove these references and the TF changes will take effect in creating a new one (https://github.com/wri/gfw-terraform-modules/blob/89b9d7d8f4ad76998e0526690140e00e212f2da4/terraform/modules/fargate_autoscaling/load_balancer.tf#L6).